### ISSUE TITLE
[Bug] MariaDB filename data too long for column

### ISSUE THREAD

**Issue by DebugDax** [zurdi15/romm#41]
When scanning /library/gb/roms/ I encountered the below error and it halted any further scanning:

```
2023-03-28T15:07:22.973406594Z sqlalchemy.exc.DataError: (mariadb.DataError) Data too long for column 'name' at row 1

2023-03-28T15:07:22.973412167Z [SQL: INSERT INTO roms (filename, filename_no_ext, r_igdb_id, p_igdb_id, r_sgdb_id, p_sgdb_id, name, r_slug, p_slug, summary, path_cover_s, path_cover_l, has_cover, size) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)]

2023-03-28T15:07:22.973418529Z [parameters: ('Mani 4 in 1 - R-Type II + Saigo no Nindou + Ganso!! Yancha Maru + Shisenshou - Match-Mania (China) (Ja).zip', 'Mani 4 in 1 - R-Type II + Saigo no Nindou + Ganso!! Yancha Maru + Shisenshou - Match-Mania (China) (Ja)', '', 33, '', '', 'Mani 4 in 1 - R-Type II + Saigo no Nindou + Ganso!! Yancha Maru + Shisenshou - Match-Mania (China) (Ja)', '', 'gb', '', '/assets/library/resources/default/cover_s.png', '/assets/library/resources/default/cover_l.png', 0, '0.21')]
```

Running via docker with mariadb as defined from the docker-compose in the repo.

**Comment by DebugDax** [comment]
I moved the files with long titles out of the folder, restarted the container, and now I cannot even restart the scan.
When clicking Scan on the frontend I get the message: "Couldn't complete scan. Something went wrong... " and in the Docker logs it shows the below:

```
2023-03-28T15:11:06.144462222Z INFO:	  [ROMM] complete scaning... 2023-03-28 15:11:06

2023-03-28T15:11:09.402536345Z INFO:     192.168.3.20:0 - "PUT /scan?overwrite=false HTTP/1.0" 500 Internal Server Error

2023-03-28T15:11:09.403992018Z ERROR:    Exception in ASGI application

2023-03-28T15:11:09.404010984Z Traceback (most recent call last):

2023-03-28T15:11:09.404017252Z   File "/usr/local/lib/python3.10/dist-packages/uvicorn/protocols/http/h11_impl.py", line 407, in run_asgi

2023-03-28T15:11:09.404024262Z     result = await app(  # type: ignore[func-returns-value]

2023-03-28T15:11:09.404029798Z   File "/usr/local/lib/python3.10/dist-packages/uvicorn/middleware/proxy_headers.py", line 78, in __call__

2023-03-28T15:11:09.404035526Z     return await self.app(scope, receive, send)

2023-03-28T15:11:09.404041126Z   File "/usr/local/lib/python3.10/dist-packages/fastapi/applications.py", line 271, in __call__

2023-03-28T15:11:09.404046971Z     await super().__call__(scope, receive, send)

2023-03-28T15:11:09.404052379Z   File "/usr/local/lib/python3.10/dist-packages/starlette/applications.py", line 118, in __call__

2023-03-28T15:11:09.404058346Z     await self.middleware_stack(scope, receive, send)

2023-03-28T15:11:09.404063790Z   File "/usr/local/lib/python3.10/dist-packages/starlette/middleware/errors.py", line 184, in __call__

2023-03-28T15:11:09.404070074Z     raise exc

2023-03-28T15:11:09.404075548Z   File "/usr/local/lib/python3.10/dist-packages/starlette/middleware/errors.py", line 162, in __call__

2023-03-28T15:11:09.404081245Z     await self.app(scope, receive, _send)

2023-03-28T15:11:09.404086719Z   File "/usr/local/lib/python3.10/dist-packages/starlette/middleware/cors.py", line 92, in __call__

2023-03-28T15:11:09.404092452Z     await self.simple_response(scope, receive, send, request_headers=headers)

2023-03-28T15:11:09.404097926Z   File "/usr/local/lib/python3.10/dist-packages/starlette/middleware/cors.py", line 147, in simple_response

2023-03-28T15:11:09.404103644Z     await self.app(scope, receive, send)

2023-03-28T15:11:09.404109017Z   File "/usr/local/lib/python3.10/dist-packages/starlette/middleware/exceptions.py", line 79, in __call__

2023-03-28T15:11:09.404114703Z     raise exc

2023-03-28T15:11:09.404139975Z   File "/usr/local/lib/python3.10/dist-packages/starlette/middleware/exceptions.py", line 68, in __call__

2023-03-28T15:11:09.404146086Z     await self.app(scope, receive, sender)

2023-03-28T15:11:09.404151377Z   File "/usr/local/lib/python3.10/dist-packages/fastapi/middleware/asyncexitstack.py", line 21, in __call__

2023-03-28T15:11:09.404156983Z     raise e

2023-03-28T15:11:09.404162013Z   File "/usr/local/lib/python3.10/dist-packages/fastapi/middleware/asyncexitstack.py", line 18, in __call__

2023-03-28T15:11:09.404167773Z     await self.app(scope, receive, send)

2023-03-28T15:11:09.404173233Z   File "/usr/local/lib/python3.10/dist-packages/starlette/routing.py", line 706, in __call__

2023-03-28T15:11:09.404178907Z     await route.handle(scope, receive, send)

2023-03-28T15:11:09.404185167Z   File "/usr/local/lib/python3.10/dist-packages/starlette/routing.py", line 276, in handle

2023-03-28T15:11:09.404190749Z     await self.app(scope, receive, send)

2023-03-28T15:11:09.404196111Z   File "/usr/local/lib/python3.10/dist-packages/starlette/routing.py", line 66, in app

2023-03-28T15:11:09.404201606Z     response = await func(request)

2023-03-28T15:11:09.404206835Z   File "/usr/local/lib/python3.10/dist-packages/fastapi/routing.py", line 237, in app

2023-03-28T15:11:09.404212453Z     raw_response = await run_endpoint_function(

2023-03-28T15:11:09.404217743Z   File "/usr/local/lib/python3.10/dist-packages/fastapi/routing.py", line 163, in run_endpoint_function

2023-03-28T15:11:09.404223411Z     return await dependant.call(**values)

2023-03-28T15:11:09.404228612Z   File "/back/./main.py", line 82, in scan

2023-03-28T15:11:09.404233924Z     platforms = data['platforms'] if data['platforms'] else fs.get_platforms()

2023-03-28T15:11:09.404239531Z   File "/back/./utils/fs.py", line 76, in get_platforms

2023-03-28T15:11:09.404245018Z     [platforms.remove(reserved) for reserved in RESERVED_FOLDERS]

2023-03-28T15:11:09.404250402Z   File "/back/./utils/fs.py", line 76, in <listcomp>

2023-03-28T15:11:09.404257271Z     [platforms.remove(reserved) for reserved in RESERVED_FOLDERS]

2023-03-28T15:11:09.404262638Z ValueError: list.remove(x): x not in list
```

**Comment by DebugDax** [comment]
> Oh dude... Can you open a new issue with that new one? Need to launch a hotfix for that ASAP.

Sure thing, opened it as #42.

**Comment by zurdi15** [comment]
Fixed in release [v1.3](https://github.com/zurdi15/romm/releases/tag/v1.3)
